staging/src/k8s.io/apiserver/pkg/storage/value/metrics.go

```diff
@@ -95,6 +95,26 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
+
+	watchCacheIncreaseTotal = metrics.NewCounter(
+		&metrics.CounterOpts{
+			Namespace:      namespace,
+			Subsystem:      subsystem,
+			Name:           "watch_cache_increase_total",
+			Help:           "Total number of watch cache capacity increase",
```

Comment:
I would like to split it by resource type (i.e. per watch cache). We probably need to pass this information from cacher though.

Reply:
Sorry, do you mean this work implemented in the follow-up pr which move these metrics to cacher.go ?

Reply:
Sorry for not being clear. What I mean is opening such PR before this one (i.e. doing this cleanup now, and then rebasing this PR on top of that one). Does that make sense? [It should be super small PR to do].

Reply:
Done, see [#90200](https://github.com/kubernetes/kubernetes/pull/90200)